documentation/administrators/multirepo/multirepo.md

```diff
@@ -0,0 +1,104 @@
+---
+title: gittuf Across Multiple Repositories
+parent: 4. Administering gittuf
+layout: default
+nav_order: 4
+permalink: /documentation/administrators/multirepo
+---
+
+# gittuf Across Multiple Repositories
+
+The original design of gittuf only allowed applying gittuf controls over a
+single Git repository. While this is sufficient for many repositories,
+organizations often have multiple (and sometimes, thousands!) of Git
+repositories to manage. Managing each repository's security policy can become an
+impossible task.
+
+Likewise, foundations and authorities (e.g. TODO) set best-practices for
+software developers to follow (e.g. multi-party code review, protecting the
+default branch of a Git repository from force-pushes, etc...).
+
+To address these limitations, gittuf was extended to allow for gittuf policies
+to **be applied over multiple repositories**. We call this **multi-repository**
+support.
+
+## Multi-Repository Architecture
+
+gittuf's multi-repository architecture can be compared to the organizational
+features provided by popular forges today (e.g. GitHub Organizations). Unlike
+these features however, gittuf is much more flexible. To understand how gittuf
+implements support for spanning across multiple repositories, let's look at the
+building blocks.
+
+### Global Rules
```

Comment:
I would move this out of multirepo. global rules are nice regardless of single vs multiple repos. It's just a policy thing imo.